glib/Gir.toml

```diff
@@ -51,17 +51,38 @@ manual = [
 name = "GLib.*"
 status = "generate"
     [[object.function]]
-    name = "clear_error"
-    #unusable
-    ignore = true
+    pattern = "(assert_warning|assertion_message|assertion_message_cmpnum|assertion_message_cmpstr|warn_message|return_if_fail_warning)"
+    ignore = true # Not useful assertions functions
     [[object.function]]
-    name = "propagate_error"
-    #empty first parameter
-    ignore = true
+    pattern = "(set_print_handler|set_printerr_handler|vasprintf|vfprintf|vprintf|vsnprintf|vsprintf|stpcpy|sprintf|snprintf|printf|printf_string_upper_bound|printerr|prefix_error|print|fprintf)"
+    ignore = true # Not useful string functions
     [[object.function]]
-    name = "set_error_literal"
-    #empty first parameter
-    ignore = true
+    pattern = "(malloc|malloc0|malloc0_n|malloc_n|memdup|memdup2|realloc|realloc_n|try_malloc||try_malloc0|try_malloc0_n|try_malloc_n|try_realloc|try_realloc_n|free)"
+    ignore = true # Not useful allocations functions
+    [[object.function]]
+    pattern = "(slice_get_config|slice_get_config_state|slice_set_config|slice_alloc|slice_alloc0|slice_copy|slice_free1|slice_free_chain_with_offset)"
+    ignore = true # Not useful slices functions
+    [[object.function]]
+    pattern = "(path_is_absolute|build_filename|build_filename_valist|build_filenamev|build_path|build_pathv|canonicalize_filename|chdir)"
+    ignore = true # Not useful paths functions, use Path/PathBuf
+    [[object.function]]
+    pattern = "(int64_equal|int64_hash|int_equal|int_hash|double_equal|double_hash|direct_equal|direct_hash)"
+    ignore = true # Not useful equal/hash functions, only useful for GHashTable
+    [[object.function]]
+    pattern = "(clear_handle_id|clear_list|clear_pointer|clear_slist|nullify_pointer)"
+    ignore = true # Not useful freeing functions
+    [[object.function]]
+    pattern = "(datalist_.*|dataset_.*)"
+    ignore = true # Not useful GData related functions
+    [[object.function]]
+    pattern = "(propagate_prefixed_error|set_error|file_error_from_errno|clear_error|propagate_error|set_error_literal|set_error)"
+    ignore = true # Not useful error handling functions
+    [[object.function]]
+    pattern = "(idle_remove_by_data|qsort_with_data)"
+    ignore = true # Unsafe functions
+    [[object.function]]
+    pattern = "(iconv|convert_with_iconv)"
+    ignore = true # not usable
     [[object.function]]
     pattern = "atomic_.+"
```

Comment:
For gio
 - `g_io_scheduler_.*` (dangerous and deprecated)
 - `g_networking_init` (useless, `std` does that already)

Otherwise seems fine but while we're at it you can also generate `GIOModuleScope` to get some more useful API generated.

Reply:
> GIOModuleScope 

It can't be auto-generated because it's a boxed type that lacks both a type function and it can't easily be manually implemented because it doesn't provide a copy function.  What should we do here?

Reply:
Ah, let's ignore that then for now until someone needs it.